doc/source/myrocks/zenfs.rst

```diff
@@ -0,0 +1,80 @@
+.. _zenfs:
+
+=========================================
+RocksDB with ZenFS
+=========================================
+
+Implemented in Percona Server for MySQL 8.0.26-16.
+
+`ZenFS <https://zonedstorage.io/projects/zenfs/>`__ is a file system plugin which places files into zones on a raw zoned block device (ZBD) using the RocksDB File System interface. ZenFS depends on ``libzbd`` and requires a Linux kernel implementation which supports MVMe Zoned namespaces. The kernel must be configured with `zone block device support enabled <https://zonedstorage.io/linux/config/#kernel-configuration>`__. 
+
+The following steps successfully run Percona Server with MySQL on the  ``--rocksdb-fs-uri=zenfs://dev:<short_block_device_name>``:
+
+.. note::
+
+    The ``<block_device_name>`` can have a short name desigination which is the ``<short_block_device_name>``. For example, the ``block_device_name`` is ``/dev/nvme1n2`` and the short name is ``nvme1n2``.
+
+#. Identify your ZBD device, ``<block_device_name>``, with ``lsblk``:
+
+    .. sourcecode:: bash
+    
+     lsblk
+
```

Comment:
Can we add some example output here? How does one identify the correct device?

Reply:
Either `lsblk -o NAME,SIZE,ZONED` or just `lsblk -z` (if `size` field is not needed) is much more informative

Reply:
@percona-ysorokin could you or maybe @inikep provide actual output from one of your test systems for reference that we could include here?

Reply:
Added.